examples/uefi/libvirt.tf

```diff
@@ -0,0 +1,43 @@
+provider "libvirt" {
+    uri = "qemu:///system"
+}
+
+resource "libvirt_network" "dev-cluster" {
+   name = "dev-cluster"
+   domain = "dev-cluster.local"
+   mode = "nat"
+   addresses = ["10.0.200.0/24"]
+}
+
+resource "libvirt_volume" "debian9-base" {
+  name = "debian9-base"
+  source = "http://localhost:8000/debian9-uefi.qcow2"
```

Comment:
If you can find an image that can be downloaded by others fine, otherwise just replace it with `debian-uefi.qcow2`. Simply drop the `http://` part